frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -22,6 +22,9 @@ import {
 } from '../extensions/console-types';
 import { StatusPopupSectionProps, StatusPopupItemProps } from '../extensions/dashboard-types';
 
+export * from '../app/components';
+export * from './common-types';
```

Comment:
@andrewballantyne from my investigation:

Both the monorepo entry point and lib-core entry point have:   
1. K8s utils (from dynamic-core-api.ts)
1. K8s type definitions
1. K8s hooks
1. useResolvedExtensions util
1. Components (missing - added)
1. Common-types (missing - added)
1. Extensions
1. useActivePerspective (through direct export in monorepo entrypoint and `core-api` in lib-core entry point)
1. perspective-context (missing - added; not required for core SDK so added to core-api)

Only in the monorepo entry point:
1. Appinitsdk
1. Reducers
1. Actions

I'll continue looking tomorrow. Let me know if anything jumps out at you from that list.

Reply:
Note - somehow `AppInitSDK` is actually being pulled into the core dist directory. Need to investigate why that is happening:

![image](https://user-images.githubusercontent.com/21317056/153525373-3fd7ce77-700e-4627-b3d2-0e8403d0e6da.png)

Reply:
Yeah, @vojtechszocs might need to look at this. I'll try to poke around and see if I can gleam what's up here.

Reply:
FWIW, it appears to be a tree shaking issue. There are imports to the index file where it is exported and thus the bundler is just grabbing everything up. It is likely part-of but not exported-out-of the Core package.

Reply:
Fixed with https://github.com/openshift/console/pull/11034/commits/d158adea1bdf2b62defb3333ad8e7d410d68485b. For now made the imports more specific, but we will need to investigate this in the future (very easy to pull it back in accidentally).